multiqc/utils/config.py

```diff
@@ -206,6 +206,27 @@ def load_sample_names(snames_file):
         logger.error("Error loading sample names file: {}".format(e))
     logger.debug("Found {} sample renaming patterns".format(len(sample_names_rename_buttons)))
 
+def load_show_hide(sh_file):
+    global show_hide_buttons, show_hide_patterns, show_or_hide
+    try:
+        with open(sh_file) as f:
+            logger.debug("Loading sample renaming config settings from: {}".format(sh_file))
+            show_hide_buttons.append("all")
```

Comment:
```suggestion
            show_hide_buttons.append("Show all")
```

Reply:
I think we should do this somewhere outside of the file parsing code. That way, if people set their patterns in a MultiQC config file instead of supplying a file the `Show all` button is still added..